### ISSUE TITLE
Use config/environment/SemanticUI for debug variable

### ISSUE THREAD

**Issue by seawatts** [Semantic-Org/Semantic-UI-Ember#177]
I think it would be nice to be able to set the debug variables in the environment file instead of creating a ember namespace

https://github.com/Semantic-Org/Semantic-UI-Ember/blob/master/addon/semantic.js#L3

**Comment by aaronbhansen** [comment]
Yeah thats some legacy code we should update. If you want to create a pull request we can merge it in.
